spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -993,6 +993,7 @@ public String getSparkUIUrl() {
   }
 
   private Results.Result interpret(String line) {
+    out.ignoreLeadingNewLinesFromScalaReporter();
```

Comment:
It's because of Scala Reporter may print some new lines in the message (not preceding new lines). https://github.com/apache/zeppelin/pull/1981/files#diff-b2cec9d09177210f5524d66dbf3fe857R59 unset this flag to print new lines from Scala Reporter. So we need to let InterpreterOutputStream know when new interpreter call starts, so it can ignore only preceding newlines from Scala Reporter, not newlines from Scala Reporter in the middle of the messages.